frontend/src/components/HtmlAnnotator.tsx

```diff
@@ -144,6 +145,14 @@ export default function HTMLAnnotator({
 		setDarkMode(window.matchMedia('(prefers-color-scheme: dark)').matches)
 	}, [setDarkMode])
 
+	useEffect(() => {
+		if (darkMode) {
+			document.documentElement.classList.add('dark')
+		} else {
+			document.documentElement.classList.remove('dark')
+		}
+	}, [darkMode])
+
```

Comment:
Thanks for the PR @xiaopeng-ye, I didn't want to do this initially because the darkmode toggle was meant to only effect the rendered HTML not the outer shell of the application.  The more I think about it though, this is probably what I would want.  As it stands today I'm setting darkMode for the outershell based on the OS setting only.  I'll merge, thanks again!

Reply:
The darkmode not working as it is supposed to i think it needs a fix